content/upbound-marketplace/providers.md

```diff
@@ -2,57 +2,100 @@
 title: "Official Providers"
 weight: 20
 ---
-Upbound creates, maintains and supports a set of Crossplane providers called *official providers*.  
+Upbound creates, maintains and supports a set of Crossplane providers called **official providers**.  
 
-## Find official providers
-Identify official providers in the marketplace with the `by Upbound` gold seal.
+Upbound builds, tests and supports official providers. The Upbound official providers are open source under the Apache 2.0 license. Upbound recommends the official providers for all deployments.
 
-{{<img src="upbound-marketplace/images/provider-by-upbound.png" alt="example official provider with a by Upbound seal" size="xtiny">}}
+Find official providers in the [Upbound Marketplace](https://marketplace.upbound.io/providers?tier=official). 
+All official providers have a purple `Official` tag.
 
-Also find official providers by filtering a [search in the marketplace](https://marketplace.upbound.io/providers?tier=official) to just `Official`.
+{{<img src="/upbound-marketplace/images/provider-by-upbound.png" alt="Official provider tag" >}}
 
-{{<img src="upbound-marketplace/images/official-provider-search-filter.png" alt="a marketplace search filter with Providers and Official filters set" size="xtiny" >}}
+## Provider families
+Upbound packages the official providers for Amazon AWS, Google GCP and Microsoft Azure as a "family." 
 
-## Install an official provider
-The [Upbound Marketplace](https://marketplace.upbound.io/) hosts official providers. 
+A provider family organizes the provider's resources into unique packages, organized by a common set of services.
 
-### Install the provider resource
-Install a provider with the creating a `Provider` Kubernetes resource with the [Up command-line]({{<ref "cli">}}) or manually.
+Provider families reduce the resources required to install and run providers and improve performance.
```

Comment:
Without stating that we are comparing them to the monolithic version of the provider, to a new user not aware of the challenges with the monolithic packages, this sentence might be redundant. Should we qualify this statement?

Reply:
How's this-

> Provider families reduce the resources required to install and run providers and improve performance over a single monolithic provider.